### DISCUSSION TITLE
Welcome to Magritte Discussions!

### DISCUSSION THREAD

**Discussion by FredDeCeuster** [Magritte-code/Magritte#17] [Show and tell]
## 👋 Welcome!
  We’re experimenting with Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by rachelglenn** [comment]
I am wondering if this code could calculate the radiation loss between two arbitrary points. Can it do that?

**Reply by FredDeCeuster** [reply]
Hi @rachelglenn,

Out of the box, unfortunately, it cannot do that.
However, I believe that all required functionality to do that is actually already in place.
Therefore, it should be possible to get something like that working either within Magritte or using its tools.
I would be happy to see if we could quickly build something (or help you build something) that would suit your needs.
For this I would need a few more details on what it is that you need, e.g. how exactly do you want to define radiation loss?
If you prefer to discuss this in private, feel free to contact me on frederik.deceuster AT kuleuven.be or the email on my profile.

Thanks for your interest!
Kind regards,

Frederik
